hs_core/templates/pages/genericresource.html

```diff
@@ -1494,10 +1494,10 @@ <h4 class="modal-title">Confirm files deletion</h4>
                 }
 
                 // If does not exist, add it
-                if (!exists && keyword[i] != "") {
+                if (!exists && keyword[i] != "" && keyword[i].length <= 100) {
```

Comment:
@pkdash 
I see. In that case, should we put this logic into a transaction to further prevent from losing old keywords if any exception occurs in update?